Cargo.toml

```diff
@@ -341,12 +341,12 @@ reth-optimism-rpc = { path = "crates/optimism/rpc" }
 reth-payload-builder = { path = "crates/payload/builder" }
 reth-payload-primitives = { path = "crates/payload/primitives" }
 reth-payload-validator = { path = "crates/payload/validator" }
-reth-primitives = { path = "crates/primitives" }
+reth-primitives = { path = "crates/primitives", default-features = false, features = ["std"] }
 reth-primitives-traits = { path = "crates/primitives-traits" }
 reth-provider = { path = "crates/storage/provider" }
 reth-prune = { path = "crates/prune/prune" }
 reth-prune-types = { path = "crates/prune/types" }
-reth-revm = { path = "crates/revm" }
+reth-revm = { path = "crates/revm", default-features = false, features = ["std", "c-kzg"] }
```

Comment:
Are we missing blst feature here?

Reply:
yeah this as well

I need to think about all of this a bit, because messing with features like this can result in giga footguns

Reply:
Yeah so in the zkVM context, I would ideally like to disable both `c-kzg` and `blst`. Ideally what would happen here is only `std` would be enabled.

Reply:
Could we just only have features = ["std"] here and then in the crates that need c-kzg/blst we just enable it there?